api/v1beta1/hcpvaultsecretsapp_types.go

```diff
@@ -32,6 +32,37 @@ type HCPVaultSecretsAppSpec struct {
 	// Destination provides configuration necessary for syncing the HCP Vault
 	// Application secrets to Kubernetes.
 	Destination Destination `json:"destination"`
+	// SyncConfig configures sync behavior from HVS to VSO
+	SyncConfig *HVSSyncConfig `json:"syncConfig,omitempty"`
+}
+
+// HVSSyncConfig configures sync behavior from HVS to VSO
+type HVSSyncConfig struct {
+	// Dynamic configures sync behavior for dynamic secrets.
+	Dynamic *HVSDynamicSyncConfig `json:"dynamic,omitempty"`
+}
+
+// HVSDynamicSyncConfig configures sync behavior for HVS dynamic secrets.
+type HVSDynamicSyncConfig struct {
+	// RenewalPercent is the percent out of 100 of a dynamic secret's TTL when
+	// new secrets are generated. Defaults to 67 percent minus jitter.
```

Comment:
Note that this is slightly different than [VaultDynamicSecret's renewalPercent](https://github.com/hashicorp/vault-secrets-operator/blob/d31cc625d686817a0f6d5f4f4ceb98b090fe8820/api/v1beta1/vaultdynamicsecret_types.go#L42-L48), since that one defaults to 67 percent *plus* to up 10% jitter.